about.html

```diff
@@ -0,0 +1,66 @@
+    </div>
+    <div class="desktop-navbar-container"><img src="images/atom.png" alt=""><nav class="desktop-nav-head-container"><a class="desktop-nav-head" href="abc.html">Home</a><h3 class="desktop-nav-head">Program</h3><h4 class="desktop-nav-head">Join</h4><h5 class="desktop-nav-head">Sponsor</h5><h6 class="desktop-nav-head">News</h6><h2 class="desktop-nav-head">ti campaign</h2></nav></div>
+    <nav class="nav-bar-about">
+        <div><span class="material-symbols-outlined" id="menu-bar2" style="font-size: 48px;color:#272A31;">menu</span></div>
+    </nav>
+    <div id='about-top-container'>
+    <h2 class="top-heading2">The ti 2022 the greatest of all.</h2>
+    <p class="about-paragraph">The major tournament of dota 2 brings all the major and minor teams under one roof and offers the chance to witness the greatest of all multiplayers clashes including some previous rivalaries and top of the teams from all corners of the world gather and compete to win the ageis and this November finally the wait would be over and make yourself ready for witnessing this mega event</p>
+<p class="about-paragraph-2">Email us for futher information about the tournament we would love to have feedback.
+</p>
+<a class="about-paragraph-3">@dota-2.org.</a>
+<h2 class="logo-heading">The logo and game dota-2 was redesigned in 2011 almost 11 years from now</h2></div>
+<div class='logo-container'><h2 class="atom-logo">The atom logo for the game that everyone wants to play</h2>
+    <div class="atom-logo-border"></div>
+<div class='dota2-logo'><img src="images/atom.png" alt=""class='dota2-logo-img'>
+    <h2 id="logo-head">18-29 nov 22 at France</h2>
+    </div>
+    
+</div>
+<div class="past-events-cotainer"><h2 class="past-events">Watch the past dota-2 events and winners</h2><div class='past-events-border'></div>
+<p  class='past-event-paragraph'>Take a look at the last 2 events that look place in france and china.</p><div class="past-event-text-container">
+    <div class="overlay">
+<div class="color-container"></div><div class="blur-img-heading">2021</div></div>
+<div class="overlay2"><div class="color-container"></div><div class="blur-img-heading">2022</div></div></div></div>
+<div class="partner-container">
+    <h2 class="partner-heading">Partner</h2>
+    <div class="partner-border"></div>
+    <div class="partners-logo-container">
+    <img src="images/water-drop.png" alt=""class='partner-logo1'>
+<img src="images/google.png" alt=""class='partner-logo1'>
+<img src="images/asus.png" alt=""class='partner-logo1'><img src="images/nvidia.png" alt=""class='partner-logo1'>
+<img src="images/energy-drink.png" alt="" class="partner-logo1"></div>
+</div>
+<div class="copy-right-container"><img src="images/atom.png" alt=""class='dota2-logo-img2'><h2 class="copy-right-heading">Dota2 ti 2022 all rights reserved.</h2></div>
```

Comment:
- [ ] Kindly add a description of the image in all of the `alt` attributes.

Reply:
fixed